src/features/auth/views/Login.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Grid, Link, Typography } from "@mui/material"
+import styled, { css } from "styled-components"
+import { Form, Formik } from "formik"
+import Button from "@/components/Button/Button"
+import loginSchema, { LoginFormValues } from "@/libs/yup/schemas/loginSchema"
+import FormikTextField from "@/components/FormikTextField/FormikTextField"
+import { Link as RouterLink } from "react-router-dom"
+import { loginUser } from "../authSlice"
+import { useAppDispatch } from "@/app/hooks"
+import { useTranslation } from "react-i18next"
+
+const Login = () => {
+  const dispatch = useAppDispatch()
+  const { t } = useTranslation(["forms", "common"])
+
+  const initialValues: LoginFormValues = {
+    email: "",
+    password: "",
+  }
+  return (
+    <StyledLogin>
+      <Box p={2}>
+        <Typography variant="h4" my={2} textAlign="center">
+          {t("common:login")}
+        </Typography>
+        <Formik
+          initialValues={initialValues}
+          onSubmit={async ({ email, password }) => {
+            await dispatch(loginUser({ email, password }))
+          }}
+          validationSchema={loginSchema}
+        >
+          {({ isSubmitting }) => (
+            <Form>
```

Comment:
OPTIONAL: You can migrate this form inside auth/components/LoginForm/LoginForm.tsx to make your Login component thinner